verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -813,7 +816,8 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                         }
                     }}
                     actions={
-                        this.props.user.roles.includes('curator')
+                        this.props.user.roles.includes('curator') ||
```

Comment:
This would also give admins the ability to verify/unverify cases (which would cause an error).

Reply:
I used the hidden attribute to hide those now.